Frameworks/include/CALayerInternal.h

```diff
@@ -29,28 +29,24 @@ class CADisplayProperties {
     BOOL hidden;
     CGRect bounds;
     CGRect contentsRect, contentsCenter;
-    CGRect contentsInset; //  Texture insets
     uint32_t contentsOrientation;
```

Comment:
>uint32_t contentsOrientation; [](start = 4, length = 29)

Just for full disclosure - contentsOrientation is not part of the CALayer API reference. It's only here because it's used by UIImage - worth a bug to investigate how to remove it cleanly